_posts/2022-05-27-gsoc-students.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Google Summer of Code Students 2022
+---
+
+We are happy to announce that MDAnalysis is hosting two [GSoC][gsoc]
+students this year -- @aya9aladdin, and @BFedder. MDAnalysis has been accepted as its own
+[organization with GSoC][mda-gsoc] for the third year running and we are grateful to Google for granting us
+the opportunity to undertake _two very exciting GSoC projects_!
+
+## Aya Mohamed Alaa: Context-aware guessers
```

Comment:
I haven't put down the lengths of each project - 12 weeks of Aya's and 6 weeks for Bjarnes I believe?

Given that we still have yet to discuss if the projects will be taken on a part-time basis, I didn't think it was appropriate to write it down here.